package.json

```diff
@@ -53,14 +53,14 @@
         " you released above, no longer work correctly."
       ],
       "deleteFromBinaryRelease": [
-        "relBinaries/i/*camlp4-*.tar.gz",
-        "relBinaries/i/esy_ocaml__slash__esy_installer-*.tar.gz",
-        "relBinaries/i/*topkg-*.tar.gz",
-        "relBinaries/i/esy_ocaml__slash__substs-*.tar.gz",
-        "relBinaries/i/*conf_which-*.tar.gz",
-        "relBinaries/i/*conf_m4-*.tar.gz",
-        "relBinaries/i/*cppo-*.tar.gz",
-        "relBinaries/i/*jbuilder-*.tar.gz"
+        "*camlp4-*",
+        "*esy_ocaml__slash__esy_installer-*",
+        "*topkg-*",
+        "*esy_ocaml__slash__substs-*",
+        "*conf_which-*",
+        "*conf_m4-*",
+        "*cppo-*",
+        "*jbuilder-*"
```

Comment:
I've changed `deleteFromBinaryRelease` to work only on gzipped tarballs as it'd be unsafe to allow removal of artefacts from installation directories — we should treat them as immutable as they can be used by `esy import-dependencies` command.

Reply:
Makes sense.

Reply:
We should merge this when `0.48` is ready to be used to cut releases? I'm testing it on some demo projects and it's working pretty well (at least on mac).